docs/3-building-blocks-of-oop-part-2/hometask/src/Weapon.ts

```diff
@@ -0,0 +1,47 @@
+import { Item } from "./Item";
+
+export abstract class Weapon extends Item {
+  protected static readonly MODIFIER_CHANGE_RATE = 0.05;
```

Comment:
according to the UML diagram should be `public static MODIFIER_CHANGE_RATE = 0.05;`